src/redux/features/observations/observationsSelectors.ts

```diff
@@ -0,0 +1,19 @@
+import { RootState } from 'src/redux/rootReducer';
+import { createSelector } from '@reduxjs/toolkit';
+
+export const selectObservationsResults = (state: RootState) => state.observationsResults?.observations;
+export const selectObservationsResultsError = (state: RootState) => state.observationsResults?.error;
```

Comment:
`state.observationResults` comes from `name: 'observationResults'` in the slice file?